src/helpers/api-functions.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  BITCOINDEV,
+  LIGHTNINGDEV,
+  type Facet,
+  type SearchQuery,
+} from "./types";
+
+const FIELDS_TO_SEARCH = ["authors", "title", "body"];
+
+type BuildQueryForElaSticClient = Omit<SearchQuery, "page"> & {
+  from: number;
+};
+
+let urlMapping = {
+  "bitcoin-dev": "https://lists.linuxfoundation.org/pipermail/bitcoin-dev",
+  "lightning-dev": "https://lists.linuxfoundation.org/pipermail/lightning-dev",
+};
+
+export const buildQuery = ({
+  queryString,
+  size,
+  from,
+  filterFields,
+  sortFields,
+  mailListType,
+}: BuildQueryForElaSticClient) => {
+  let baseQuery = {
+    query: {
+      bool: {
+        must: [] as any[],
+        should: [] as any[],
+        filter: [
+          {
+            terms: {
+              "domain.keyword": [
+                urlMapping["bitcoin-dev"],
+                urlMapping["lightning-dev"],
+              ],
+            },
+          },
+        ] as any[],
+      },
+    },
+    sort: [] as any[],
+    aggs: {
+      authors: {
+        terms: {
+          field: "authors.keyword",
+          size: 15,
```

Comment:
if this is always fixed, lets make it a const in a constant file. I think we have one in the repo